WordPress/src/main/java/org/wordpress/android/ui/posts/editor/AztecEditorFragmentStaticWrapper.kt

```diff
@@ -0,0 +1,19 @@
+package org.wordpress.android.ui.posts.editor
+
+import android.content.Context
+import dagger.Reusable
+import org.wordpress.android.editor.AztecEditorFragment
+import javax.inject.Inject
+
+/**
+ * Injectable wrapper around AztecEditorFragment.
+ *
+ * AppPrefs interface contains some static methods, which make the client code difficult to test/mock. Main purpose of
+ * this wrapper is to make testing of these static methods easier.
+ *
+ */
+@Reusable
+class AztecEditorFragmentStaticWrapper @Inject constructor(private val appContext: Context) {
```

Comment:
I've created [something similar in my PR](https://github.com/wordpress-mobile/WordPress-Android/blob/3e0a4e3b9399c7bd459c6e00329f5526d1cc29c9/WordPress/src/main/java/org/wordpress/android/ui/posts/AztecEditorWrapper.kt), we need to think about it when we merge the PRs

Reply:
The PR is already merged. Do you think you can now merge the 2 classes? :-)

Reply:
@planarvoid Hmm, I checked develop yesterday and I couldn't  find `AztecEditorWrapper` there. It seems we merged the PR into the `master_edit_post_activity_refactoring` branch but this PR targets develop.